core/assets/jsons/save.json

```diff
@@ -49,80 +49,129 @@
     "path": "levels/shooter-tutorial.json"
   },
   {
-    "name": "level 5",
+    "name": "Cave level 8",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
     "path": "levels/easyLevel.json"
   },
   {
-    "name": "level 6",
+    "name": "Cave level 9",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
     "path": "levels/mediumLevel.json"
   },
   {
-    "name": "level 7",
+    "name": "Cave level 10",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
     "path": "levels/hardLevel.json"
   },
   {
-    "name": "level 8",
+    "name": "Cave level 11",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
-    "path": "levels/enemySurroundingPlayer.json"
+    "path": "levels/enemySurroundingPlayerUpdated.json"
   },
   {
-    "name": "level 9",
+    "name": "Cave level 12",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
-    "path": "levels/forceSneak.json"
+    "path": "levels/forceSneakUpdated.json"
   },
   {
-    "name": "level 10",
+    "name": "Cave level 13",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
-    "path": "levels/shooterAndPatrol.json"
+    "path": "levels/Jalil-11.json"
   },
   {
-    "name":       "level 11",
+    "name":       "Cave level 14",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
-    "path":       "levels/maxLevel.json"
+    "path":       "levels/medium.json"
   },
   {
-    "name": "level 12",
+    "name": "Cave level 15",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
-    "path": "levels/onlypatrol.json"
+    "path": "levels/Jalil-12.json"
```

Comment:
Enemy in conveyor belt gets stuck after you throw a flare, maybe make the conveyor belt path a tad wider